commandline/src/main/java/tech/pegasys/web3signer/commandline/Web3SignerBaseCommand.java

```diff
@@ -82,11 +85,22 @@ public class Web3SignerBaseCommand implements Config, Runnable {
 
   @Option(
       names = {"--key-store-path"},
-      description = "The path to a directory storing toml files defining available keys",
+      description = "The path to a directory storing yaml files defining available keys",
       paramLabel = DefaultCommandValues.MANDATORY_PATH_FORMAT_HELP,
       arity = "1")
   private Path keyStorePath = Path.of("./");
 
+  @Option(
+      names = {KEY_STORE_CONFIG_FILE_SIZE_OPTION_NAME},
+      description =
+          "The key store configuration file size in bytes. Useful when loading a large number of configurations from "
+              + "the same yaml file. Defaults to (${DEFAULT-VALUE}) 100MB",
+      paramLabel = "<NUMBER>",
+      defaultValue = "104857600",
+      converter = YamlMapperCliConverter.class,
```

Comment:
What happens if you set this with a config file instead? Or if you don't set it at all - how/when does the YamlMapper get created in those situations?

Reply:
Config file and cli both eventually ends up here. If we don't set it either via config file and cli, the default value gets assigned.

Reply:
nice!